modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java

```diff
@@ -157,6 +157,10 @@ private String _getDefaultLanguageId() {
 	}
 
 	private String _getModule() {
+		if (DataLayoutTaglibUtil.isUsingNewFormProvider()) {
```

Comment:
I placed the debugger here to see the value of `DataLayoutTaglibUtil.isUsingNewFormProvider()`, it comes "false" but I created the config changing to "true"

Reply:
Solved, I included the ff conditional if any modules was given to the data-layout-builder ;)

```java
	private String _getModule() {
		if (Validator.isBlank(getModule())) {
			if (DataLayoutTaglibUtil.isUsingNewFormProvider()) {
				return "data_layout_builder/new-js/App.es";
			}
			return "data_layout_builder/js/App.es";
		}
		return getModule();
	}
```

Reply:
I needed to make another modification, replacing the `configurationPid` of the ConfigurationActivator class to the already Configuration class. :)

Reply:
Thanks @jeyvison <3